rfc/updates.md

```diff
@@ -194,6 +194,51 @@ Note that:
   Images are specified by the CLI upon loading the Helm chart, by inspeciting `constellation-conf.yaml`.
   Deployment variations could be introduced into the Helm charts if they become necessary in the future.
 
+### CLI commands
+
+`constellation upgrade` has 2 sub commands:
+
+* `constellation upgrade check`
+* `constellation upgrade apply`
+
+When `constellation upgrade check` is called it checks if the current CLI embeds helm charts and kubernetes components which are newer than the ones defines in the `constellation-config.json`. If this is the case, it prints a detailed list of all the components that will be updated. Moreover, it checks via the update API (see: rfc/update-api.json) for new image patch versions as they are always compatible with inside a minor version.
+
+Lastly, it uses the update API (see: rfc/update-api.json) if there is a newer CLI version available. If this is the case, it will print the latest version and omit ony of the points above since. If this version is more than +1 minor version apart it will also show the latest CLI of the next minor version, and suggest a way to download it. Note that if there are still e.g. microservice updates needed with the current CLI, we need to promt the user to first install those before continuing with the next minor release.
+
+We also print `In newer CLI versions there are even newer versions available.` if e.g. there is a newer patch version of Kubernetes available in one of the proposed minor versions.
```

Comment:
Wouldn't this required quite a complicated lookup, or at least an extension of the current update API, to resolve which versions will be available with another CLI version?

Reply:
I don't think so. For the microservices we should just be able to get the latest version from the upgrade API and print this notification if the latest does not match the one embedded in the CLI.

Reply:
That makes sense :).